doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -34,6 +34,10 @@ A more complete set of build instructions are included here for multiple platfor
 - [macOS :apple:](#macos)
 - [ARM :iphone:](#arm)
 
+User documentation for the latest release of Eclipse OpenJ9 is available at the [Eclipse Foundation](https:/www.eclipse.org/openj9/docs).
```

Comment:
The https link is missing a slash after `https:`, doesn't link to the correct place. Same for the V8 docs.

Reply:
Fixed! Thanks